src/build/languages.js

```diff
@@ -1,5 +1,4 @@
-module.exports = {
+exports.languages = {
```

Comment:
We aren't actually export a `module` per say, but a helper object, which falls into the category of the `exports` case.